app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductBrowsingListActivity.java

```diff
@@ -222,7 +224,7 @@ private void loadData(boolean isResponseOk, Search response) {
         if (isResponseOk) {
             mCountProducts = Integer.parseInt(response.getCount());
             if (pageAddress == 1) {
-                countProductsView.append(" " + NumberFormat.getInstance(getResources().getConfiguration().locale).format(Long.parseLong(response.getCount()
+                countProductsView.setText(getResources().getString(R.string.number_of_products)+ " " + NumberFormat.getInstance(getResources().getConfiguration().locale).format(Long.parseLong(response.getCount()
```

Comment:
This is the only piece of code that needs to be changed to resolve this. Please ensure that the code is tested throughout the app lifecycle to ensure that the use of the `setUp` variable does not prevent this from handing `onResume()` and the rest of the app lifecycle